### PR TITLE
Bybit: Fix UpdateAccountInfo coin balance issue, improve GetWalletBalance and enhance test coverage

### PR DESCRIPTION
**Pull request by thrasher-**
# PR Description

Thanks to Lord Romano for identifying this issue! This PR addresses a coin balance problem in UpdateAccountInfo, improves GetWalletBalance fields, enhances test coverage and makes Bybit use the new test exchange helpers.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How has this been tested

- [x] go test ./... -race
- [x] golangci-lint run

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally and on Github Actions with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: exchanges/bybit/bybit_types.go [hunk 1]
```diff
@@ -881,27 +881,30 @@ type WalletBalance struct {
...
+			AvailableToWithdraw     types.Number  `json:"availableToWithdraw"`
+			AvailableBalanceForSpot types.Number  `json:"free"`
+			TotalOrderIM            types.Number  `json:"totalOrderIM"`
+			Equity                  types.Number  `json:"equity"`
+			Locked                  types.Number  `json:"locked"`
+			MarginCollateral        bool          `json:"marginCollateral"`
+			SpotHedgingQuantity     types.Number  `json:"spotHedgingQty"`
+			TotalPositionMM         types.Number  `json:"totalPositionMM"`
+			USDValue                types.Number  `json:"usdValue"`
+			UnrealisedPnl           types.Number  `json:"unrealisedPnl"`
```
Comment (gloriousCode) [inline exchanges/bybit/bybit_types.go:901]:
```suggestion
			UnrealisedPNL           types.Number  `json:"unrealisedPnl"`
```
Huge issue, but puts it in line with everything else

**Review by gloriousCode — CHANGES_REQUESTED** [review]
Thanks! Endpoints worked, but I have no money there so its an empty range 😭 
All API properties accounted for. Just one massive showstopper in line with CrowdStrike

**Review by gloriousCode — APPROVED** [review]
[tACK!](https://www.youtube.com/watch?v=lA-UBd6w9Yo)

**Review by shazbert — APPROVED** [review]
tACK!

**Pull request merged**